src/TaskBoard.js

```diff
@@ -18,6 +21,10 @@ export default function TaskBoard(){
        done: false
      }
     ]);
+    e.target.reset(); //This works
+    //this is still not working after lifting state
+    // setTaskInupt("");
+    // setAsignee("");
     
```

Comment:
@Tom-Sarfo always remove commented code before pushing

Reply:
Okay thank you